docs/admin/serving/webhook-customizations.md

```diff
@@ -2,6 +2,8 @@
 title: "Exclude namespaces from the Knative webhook"
 weight: 70
 type: "docs"
+aliases:
+  - /docs/serving/webhook-customizations
```

Comment:
Again, this actually applies to both serving and eventing, and maybe deserves a "common" configuration options section.

Reply:
awesome, thanks for letting me know